keras_nlp/tokenizers/unicode_character_tokenizer_test.py

```diff
@@ -290,12 +334,39 @@ def test_config(self):
             "input_encoding": "UTF-16",
             "output_encoding": "UTF-16",
             "trainable": True,
+            "vocabulary_size": None,
         }
         self.assertEqual(
             tokenize_different_encoding.get_config(),
             exp_config_different_encoding,
         )
 
+        tokenize_different_vocabular_size = UnicodeCharacterTokenizer(
```

Comment:
You could probably just move setting `vocabulary_size` up into the first example in this by setting it in `tokenizer` and `exp_config`. And ditch this extra case here.

We want to keep this test from exploding inside, and I don't think we should try to test every permutation here.